web/src/drafts.ts

```diff
@@ -196,6 +199,7 @@ export const draft_model = (function () {
         // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
         delete drafts[id];
         save(drafts);
+        update_compose_draft_count();
```

Comment:
Should `editDraft` call this too, in case a draft's recipient changes?

Reply:
I moved this into `save` to be alongside `set_count` -- no repetitive calls now!